db/views/analytics_client_geolocations_v01.sql

```diff
@@ -0,0 +1,18 @@
+SELECT
+    id,
+    client_id,
+    source_type,
+    source_id,
+    located_on,
+    lat,
+    lon,
+    -- collected_by,
+    -- processed_at,
+    created_at,
+    updated_at,
+    enrollment_id,
```

Comment:
I think we want to drop `enrollment_id`, from my understanding this field was legacy for some imported warehouse data, but doesn't get set anymore. If the location is tied to an enrollment, it is related through `source_id`/`source_type`.

```suggestion
```